src/components/common/notification-center.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
+  import { onMount } from 'svelte';
+  import Dropdown from 'ui/dropdown.svelte';
+  import Button from 'ui/button.svelte';
+  import * as api from '@/utils/api.js';
+
+  export let notifications = null;
+  $: unread = notifications != null && notifications.some(notification => !notification.is_read);
+
+  onMount(async () => {
+    try {
+      let resp = await api.get('/notifications');
+      if (resp.status === 200) {
+        notifications = await resp.json();
+      }
+    } catch (e) { console.error(e); }
+  });
+</script>
+
+<Dropdown
+  dropdownclass="notification-center"
+  handleclass="round"
+  chevron={false}
+  nowrap
+  let:toggle={toggle}
+>
+  <div slot="label" class="badge" class:hidden={!unread}>
+    <svg src="images/icons/bell.svg" class="icon" />
+  </div>
+  <Button isNormal isRound classname="close" on:click={toggle}>
+    <svg src="images/icons/x.svg" class="icon" />
+  </Button>
+  <div class="title">Notifications</div>
+  {#if notifications == null}
+    <div class="empty small">
+      <div class="icon">
+        <div class="lds-ellipsis"><div></div><div></div><div></div><div></div></div>
```

Comment:
Wtf?
Loop, may be?

Reply:
This is copy-pasted code from https://loading.io/css/, I really don't see a reason to change it

Reply:
Is the way it looks not enough reason?

Reply:
There isn't even a built-in `range` in JS as in Python. Do you really want me to import a generator from utils just because you don't like to see four plain divs in a row?